src/sass/_all.scss

```diff
@@ -0,0 +1,310 @@
+        a{
+          font-family: Arial;
+          font-size: 15px;
+          line-height: 160%;
+          color: $color-header-footer-li;
+        }
+
+        .active{
+          border-bottom: 1px solid $color-header-footer-primary;
+          color: #FAFAFA;
+        }
+      }
+    }
+}
+
+.startscreen{
+  width: 1200px;
+  display: flex;
+  text-align: left;
+  justify-content: space-between;
+  margin: 180px 40px auto 40px;
+
+  .textstart{
+    max-width: 460px;
+    margin: 163px 42px 243px 0;
+
+    h2{
+      color: $color-header-footer-h;
+      font-size: 44px;
+      line-height: 130%;
+    }
+
+    p{
+      color: $color-header-footer-li;
+      margin: 42px auto;
+    }
+  }
+
+  .startbutton{
+    a{
+      color: $color-buttons;
+    }
+
+    background-color: $background-buttons;
+    border-radius: $border-radius;
+    text-align: center;
+    max-width: 207px;
+    padding: 15px 45px;
+  }
+}
+
+.container{
+  margin: 0 auto;
+  max-width: 850px;
+}
+
+.containerAbout {
+  height: 588px;
+}
+
+.about{
+  background-color: $background-about-help;
+  display: flex;
+  text-align: center;
+  justify-content: space-between;
+  
+  img {
+    margin: 80px auto;
+  }
+  
+  .text-about {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 430px;
+    
+    h3 {
+      width: 370px;
+    }
+
+    p{
+      text-align: justify;
+      margin: 25px auto;
+      color: $color-about-p;
+    }
+  }
+}
+
+.pets{
+  background-color: $background-pets-donation;
+  background-repeat: repeat;
+  overflow: hidden;
+  height: 877px;
+
+  h3 {
+    text-align: center;
+    width: 400px;
+    margin: 80px auto 60px auto;
+  }
+}
+.pets-grid {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+
+  button {
+    border: 2px solid $background-buttons;
+    border-radius: 100px;
+    height: 52px;
+    width: 52px;
+    padding: 0 17px;
+    align-items: center;
+  }
+  
+  .hover:hover{
+    background-color: $background-buttons;
+    cursor: pointer;
+    transition-duration: .2s;
+  }
+}
+
+.pets-card {
+  background-color: $background-about-help;
+  margin: auto 45px;
+  
+  p{
+    margin: 30px auto;
+  }
+  .pets-buttons {
+    border: 2px solid $background-buttons;
+    border-radius: $border-radius;
+    padding: 15px 45px;
+    margin: 30px 41.5px;
+
+    a {
+      font-size: 17px;
+      color: $color-buttons;
+    }
+  }
+}
+
+.pets-button {
+  a{
+    color: $color-buttons;
+  }
+
+  font-size: 17px;
+  background-color: $background-buttons;
+  border-radius: $border-radius;
+  text-align: center;
+  max-width: 261px;
+  padding: 15px 45px;
+  margin: 60px auto;
+}
+
+.help-content {
+  height: 611px;
+  margin-top: 80px;
+  
+  h3 {
+    text-align: center;
+    width: 308px;
+    margin: 0 auto;
+    
+  }
+  .help-icon {
+    display: flex;
+    flex-wrap: wrap;
+    width: 1026px;
+    align-items: center;
+    justify-content: space-around;
```

Comment:
I think the icons will look more centrilized  with `justify-content: center;`  and equivalent margin-left and margin-right for  each figure